.neuro/live.yml

```diff
@@ -0,0 +1,50 @@
+kind: live
+title: Fooocus
+
+defaults:
+  life_span: 5d
+  preset: a100x1
+
+images:
+  fooocus:
+    ref: ghcr.io/neuro-inc/fooocus:latest
+  fooocus_apolo:
+    ref: image:${{ flow.project_id }}
+    context: ${{ flow.workspace }}
+    dockerfile: ${{ flow.workspace }}/Dockerfile
+    build_preset: cpu-medium
```

Comment:
we could drop build_preset, since this preset might not be available in cluster

Reply:
Can I somehow suggest a minimum ram requirement for this build job? Because it requires at least 17GB and failed when I used cpu-small